checkov/terraform/checks/resource/alicloud/KMSKeyIsEnabled.py

```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from typing import Any
+
+
+class KMSKeyIsEnabled(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure KMS Keys are enabled"
+        id = "CKV_ALI_26"
```

Comment:
```suggestion
        id = "CKV_ALI_28"
```
looks like 26 is for a Kubernetes check 😄